src/SixLabors.Fonts/Glyph.cs

```diff
@@ -9,11 +9,19 @@ namespace SixLabors.Fonts
     /// <summary>
     /// A glyph from a particular font face.
     /// </summary>
-    internal readonly struct Glyph
+    public readonly struct Glyph
     {
         private readonly GlyphInstance instance;
         private readonly float pointSize;
 
+        /// <summary>
+        /// Gets the glyph instance.
+        /// </summary>
+        /// <value>
+        /// The glyph instance.
+        /// </value>
+        public GlyphInstance GlyphInstance => this.instance;
```

Comment:
Glyph prefix is redundant. 

```
public GlyphInstance Instance => this.instance;
```

Reply:
Should `FontInstance` in Font also be renamed?

Reply:
I would say so yes 👍

Reply:
Did the required change